keps/sig-node/1769-memory-manager/README.md

```diff
@@ -512,63 +576,83 @@ you need any help or guidance.
 
 _This section must be completed when targeting alpha to a release._
 
-* **How can this feature be enabled / disabled in a live cluster?**
-  - [ ] Feature gate (also fill in values in `kep.yaml`)
+###### How can this feature be enabled / disabled in a live cluster?
+
+  - [X] Feature gate (also fill in values in `kep.yaml`)
     - Feature gate name: MemoryManager
     - Components depending on the feature gate: kubelet
     - Will enabling / disabling the feature require downtime of the control
       plane? No
     - Will enabling / disabling the feature require downtime or reprovisioning
       of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled). Yes, it uses a feature gate.
 
-* **Does enabling the feature change any default behavior?**
+###### Does enabling the feature change any default behavior?
+
 Yes, the admission flow changes for a pod in Guaranteed QoS class. With the Memory Manager, Topology Manager also takes into account allocatable memory (and hugepages) to either admit a pod to the node or reject it. 
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?** Yes, it uses a feature gate.
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, the feature is using `MemoryManager` feature gate and can be disabled.
+Disabling the feature gate requires kubelet restart for changes to take effect.
+In case no workloads are running on the node, or consuming memory, disabling the 
+feature should be seamless.
+
+In case there are running workloads which are requested for memory, it's the 
+cluster admin's responsibility to drain the node, prior to the feature disablement.
```

Comment:
What's the failure mode if that doesn't happen?

Reply:
Added to the text, thanks.